css/_filmstrip.scss

```diff
@@ -121,4 +121,13 @@
             }
         }
     }
+
+    /**
+     * Style the film strip videos in film-strip-only mode.
+     */
+    &__videos-filmstripOnly {
+        margin-top: auto;
+        margin-bottom: auto;
+        padding-right: $defaultToolbarSize;
+    }
```

Comment:
This new css class isn't connected to the variable name change, it's only used in the next commits.